aws/modules/infrastructure_modules/vpc/vpc.tf

```diff
@@ -34,6 +34,7 @@ module "vpc" {
     "isLambda"                              = "false"
     "isFirewall"                            = "false"
     "isDB"                                  = "false"
+    "Name"                                  = "${var.vpc_name}-private"
```

Comment:
Don't these tags already exist as a result of:
```tf
private_subnet_suffix  = "private"
database_subnet_suffix = "database"
```

Reply:
Unfortunately it doesnt. I can have a look into the logic and try to figure out but may be later
![Screenshot 2024-06-12 at 16 03 28](https://github.com/Ensono/stacks-terraform/assets/78567578/3c7d547a-32d4-4844-b7bd-b26463dd849e)

Reply:
done